tidy3d/plugins/adjoint/components/simulation.py

```diff
@@ -317,6 +319,8 @@ def plot_eps(
         alpha: float = None,
         source_alpha: float = None,
         monitor_alpha: float = None,
+        hlim: Tuple[float, float] = None,
+        vlim: Tuple[float, float] = None,
```

Comment:
we should also pass these into the `sim.plot()` and `sim.plot_eps()` calls in these methods. basically, this is making simulation plotting work in the `JaxSimulation` subclass of `Simulation`.